developer/src/kmc-kmw/test/test-compiler-manual.ts

```diff
@@ -26,18 +26,19 @@ if(!await kmnCompiler.init(callbacks)) {
   process.exit(1);
 }
 
-// TODO: runToMemory, add option to kmxCompiler to store debug-data for conversion to .js (e.g. store metadata, group readonly metadata, etc)
-if(!kmnCompiler.run(infile, outfile, {
+let result = kmnCompiler.runCompiler(infile, outfile, {
   shouldAddCompilerVersion: false,
-  saveDebug: true,
+  saveDebug: true,  // TODO: we should probably use passed debug flag
```

Comment:
Just pointing this one out.  I'm not sure if it's related to the following or not:

> kmc-kmw refactor to invert dep on kmc-kmn

Reply:
Yep, will be sorted out when I add proper tests for kmw compiler.